src/Microsoft.DncEng.Configuration.Extensions/Microsoft.DncEng.Configuration.Extensions.csproj

```diff
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFramework></TargetFramework>
```

Comment:
Not really sure why this clear of `TargetFramework` prop is needed here to make the build produce both net6.0 and netstandard2.0 assemblies. Guessing its because the prop is defined in root `Directory.Build.props`?

Reply:
Yes. It's a bit stupid and you can't have both